features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx

```diff
@@ -0,0 +1,297 @@
+                        </Step>
+
+                        <Step>
+                            <StepLabel>
+                                <FormattedMessage
+                                    id='Apis.Create.GraphQL.ApiCreateGraphQL.wizard.two'
+                                    defaultMessage='Create API'
+                                />
+                            </StepLabel>
+                        </Step>
+                    </Stepper>
+                )}
+                {wizardStep === 1 && (
+                    <Stepper activeStep={1}>
+                        <Step>
+                            <StepLabel>
+                                <FormattedMessage
+                                    id='Apis.Create.GraphQL.ApiCreateGraphQL.wizard.one'
+                                    defaultMessage='Provide GraphQL'
+                                />
+                            </StepLabel>
+                        </Step>
+
+                        <Step>
+                            <StepLabel>
+                                <FormattedMessage
+                                    id='Apis.Create.GraphQL.ApiCreateGraphQL.wizard.two'
+                                    defaultMessage='Create API'
+                                />
+                            </StepLabel>
+                        </Step>
+                    </Stepper>
+                )}
+            </Paper>
+
+            <Grid container spacing={3}>
+                <Grid item md={12} />
+                <Grid item md={1} />
+                <Grid item md={11}>
+                    {wizardStep === 0 && (
+                        <ProvideGraphQL
+                            onValidate={handleOnValidate}
+                            apiInputs={apiInputs}
+                            inputsDispatcher={inputsDispatcher}
+                        />
+                    )}
+                    {wizardStep === 1 && (
+                        <DefaultAPIForm onValidate={handleOnValidate} onChange={handleOnChange} api={apiInputs} />
+                    )}
+                </Grid>
+                <Grid item md={1} />
+                <Grid item md={9}>
+                    <Grid container direction='row' justify='space-between'>
+                        <Grid item>
+                            {wizardStep === 0 && (
+                                <Link to='/apis/'>
+                                    <Button variant='outlined'>
+                                        <FormattedMessage
+                                            id='Apis.Create.GraphQL.ApiCreateGraphQL.cancel'
+                                            defaultMessage='Cancel'
+                                        />
+                                    </Button>
+                                </Link>
+                            )}
+                            {wizardStep === 1 && <Button onClick={() => setWizardStep(step => step - 1)}>Back</Button>}
```

Comment:
'Back' is a button text? if so don't we need to localize it?

Reply:
resolved by #8330